app/customer.py

```diff
@@ -0,0 +1,53 @@
+from math import sqrt
+
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(self, name: str, products: dict,
+                 location: list, money: int, car: Car) -> None:
+        self.name = name
+        self.products = products
+        self.location = location
+        self.money = money
+        self.car = car
+
+    def calculate_distance(self, shop: list) -> float:
```

Comment:
The `calculate_distance` method expects a `shop` parameter as a list, but it should be a `Shop` object to access its location. Ensure that the `shop` parameter is used correctly as a `Shop` object.